app/service/rest_svc.py

```diff
@@ -124,11 +124,12 @@ def __init__(self):
         return dict(contacts=self.get_service('contact_svc').report.get(contact.get('contact'), dict()))
 
     async def update_agent_data(self, data):
+        if 'paw' not in data:
```

Comment:
why this file change? any difference?

Reply:
Previously, if you did a PUT specifying a paw that did not exist, it would change the global settings.  I ran into it when I was testing the agent termination code and forgot to update the paw between requests -- the result was that I tried to kill the same agent twice and then all new agents would kill themselves by default. 

I think this is more what the original functionality was going for (by leaving paw out, you change the default settings for all agents).